mLRS/CommonTx/crsf_interface_tx.h

```diff
@@ -200,6 +200,17 @@ void tTxCrsf::parse_nextchar(uint8_t c)
             cnt = 0;
             frame[cnt++] = c;
             state = STATE_RECEIVE_CRSF_LEN;
+            if (discarded) {
```

Comment:
this is only for taking statistics, right?

if so I would disable it with no debug

also, this code piece appears in few placed, can't it be a function or a macro?